staging/cse/windows/provisioningscripts/cleanupnetwork.ps1

```diff
@@ -1,5 +1,17 @@
+# The cleanup process for `HNS Network` and `CNI data` should be consistent.
+
 $Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
 
+$global:$IsSkipCleanupNetwork = [System.Convert]::ToBoolean($Global:ClusterConfiguration.Services.IsSkipCleanupNetwork)
+
+filter Timestamp { "$(Get-Date -Format o): $_" }
+
+function Write-Log($message) {
```

Comment:
cleanupnetwork.ps1 is executed in staging/cse/windows/provisioningscripts/windowsnodereset.ps1. could we wrapper this as a function and import it in staging/cse/windows/provisioningscripts/windowsnodereset.ps1 so do not use a new log file?

Reply:
redirect to nodereset.log